src/test/groovy/com/gooddata/md/report/TotalTest.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.md.report
+
+import spock.lang.Specification
+
+import static com.gooddata.md.report.Total.AVG
+import static com.gooddata.md.report.Total.MAX
+import static com.gooddata.md.report.Total.MED
+import static com.gooddata.md.report.Total.MIN
+import static com.gooddata.md.report.Total.NAT
+import static com.gooddata.md.report.Total.SUM
+
+class TotalTest extends Specification {
+    def "should for Total.of(...) throw exception on invalid name"() {
```

Comment:
please reword the test summary into the shape of sentence

Reply:
done